mlflow/johnsnowlabs/__init__.py

```diff
@@ -367,11 +382,11 @@ def log_model(
             remote_model_path=remote_model_path,
             store_license=store_license,
         )
-        mlflow.tracking.fluent.log_artifacts(tmp_model_metadata_dir, artifact_path)
+        mlflow.tracking.fluent.log_artifacts(tmp_model_metadata_dir, name)
         mlflow.tracking.fluent._record_logged_model(mlflow_model)
         if registered_model_name is not None:
             mlflow.register_model(
-                f"runs:/{run_id}/{artifact_path}",
+                f"runs:/{run_id}/{name}",
```

Comment:
The model should be registered using a `models:/` URI, not a `runs:/ URI`, since the model isn't logged to a Run.